plugins/uefi-dbx/meson.build

```diff
@@ -0,0 +1,60 @@
+cargs = ['-DG_LOG_DOMAIN="FuPluginUefiDbx"']
+
+shared_module('fu_plugin_uefi_dbx',
+  fu_hash,
+  sources : [
+    'fu-plugin-uefi-dbx.c',
+    'fu-uefi-dbx-common.c',
+    'fu-uefi-dbx-file.c',
+  ],
+  include_directories : [
+    root_incdir,
+    fwupd_incdir,
+    fwupdplugin_incdir,
+  ],
+  install : true,
+  install_dir: plugin_dir,
+  link_with : [
+    fwupd,
+    fwupdplugin,
+  ],
+  c_args : cargs,
+  dependencies : [
+    plugin_deps,
+  ],
+)
+
+# only install this copy if the user has not specified another location
+# downloaded from https://uefi.org/revocationlistfile on 2020-05-06
+if efi_dbxdir == ''
```

Comment:
Since the build process doesn't normally use the web, I think it might be worth adding to one of the CI tasks to checks if this file has changed and makes noise for it to be updated

Reply:
tbh, I don't really want to ship the file at all...

Reply:
Thinking about it more rationally, I think we have to ship a fallback, otherwise the HSI is going to be incomplete on every non-mainstream Linux. I really want this to "just work" without having to hunt things down. I've added a CI test to check if the file is up to date.